assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/table/styles.scss

```diff
@@ -16,12 +16,4 @@
 			}
 		}
 	}
-
-	&-field {
-		display: flex;
-		&__before {
-			line-height: var(--ee-icon-button-size);
-			margin: 0 var(--ee-margin-smaller) 0 var(--ee-margin-small);
-		}
-	}
```

Comment:
this was collocated to `TicketPriceField`, table is too high in the tree

Reply:
ya smart 👍